apps/music/README.md

```diff
@@ -0,0 +1,91 @@
+# Music
+
+The Music app is one of the Gaia's core media apps. It's capable of:
+
+- Displays the user's collection by **mix**.
+- Displays the user's collection by **playlist**.
+- Displays the user's collection by **album**.
+- Displays the user's collection by **artist**.
+- Displays the user's collection by **title**.
+- Allows the user to **play** the songs.
+- Allows the user to **share** the songs as ringtones.
+- Allows the user to **view** the downloaded audio files.
+- Allows the user to **search** the songs.
+
+It scans the internal/external storages on the device. Copying your songs into the internal storage or sd card could both be found then displayed in the Music app. It recognized the [Extensions](https://dxr.mozilla.org/mozilla-central/source/toolkit/content/devicestorage.properties) that the [DeviceStorage](https://developer.mozilla.org/en-US/docs/Web/API/Device_Storage_API) api knows about.
+
+Testing Music app will need some contents so copying your audio files into:
+
+- [Firefox OS Simulator](https://developer.mozilla.org/en-US/docs/Tools/Firefox_OS_Simulator) or [Firefox Developer Edition](https://developer.mozilla.org/en-US/Firefox/Developer_Edition): the default music folder under your home directory(slightly different among Ubuntu, Windows and OSX).
+- [Developer phones](https://developer.mozilla.org/en-US/Firefox_OS/Phone_guide): Settings app > Enable the USB Storage > the storages will be mounted to your computer.
+
+## App
+
+[App](js/music.js) object is the first thing you should know about this app. It's the entry of Music and setups the basis, like:
+
+- **Start Mode**: Music app has there start modes, there are:
+  - *Regular*: every time we launch the Music app from the homescreen icons, it's the regular start mode.
+  - *Pick*: the other apps, such as SMS or Email needs the Music app to provide the audio attachments, Music app will be launched as open [Activity](https://developer.mozilla.org/en-US/docs/Web/API/Web_Activities).
+  - **Open*: this start mode is actually not setup by the App object, it's handled by separated files, which are [open.html](open.html) and [open.js](js/open.js).
```

Comment:
@jimporter Actually the extra asterisk is to emphasize the Open start-mode is NOT setup by the App object, maybe I should use two asterisks to quote the `not` word so it will not looks like a markup typo.